data-pipeline/fetch_data.py

```
@@ -39,8 +39,14 @@
 class WaterQualityDataFetcher:
     """Main class for fetching water quality data from various sources"""
     
-    def __init__(self, db_path: str = "water_quality_data.db"):
-        self.db_path = db_path
+    def __init__(self, db_path: str = None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)